script.module.pymysql/addon.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pymysql"
+       name="pymysql"
+       version="0.8.0"
+       provider-name="smitchell6879">
+  <requires>
+    <import addon="xbmc.python" version="2.24.0"/>
```

Comment:
Krypton repository requires `xbmc.python` 2.25.0

Reply:
This can be bumped up I just double checked it will work with 2.25.